lib/tls.js

```diff
@@ -15,6 +15,13 @@ exports.CLIENT_RENEG_WINDOW = 600;
 exports.SLAB_BUFFER_SIZE = 10 * 1024 * 1024;
 
 exports.DEFAULT_CIPHERS = [
+  'ECDHE-RSA-AES128-GCM-SHA256',
+  'ECDHE-ECDSA-AES128-GCM-SHA256',
+  'ECDHE-RSA-AES256-GCM-SHA384',
+  'ECDHE-ECDSA-AES256-GCM-SHA384',
+  'DHE-RSA-AES128-GCM-SHA256',
+  'DHE-DSS-AES128-GCM-SHA256',
```

Comment:
Mozilla's logic is https://wiki.mozilla.org/Security/Server_Side_TLS#Prioritization_logic:

> ECDHE+AESGCM ciphers are selected first. These are TLS 1.2 ciphers and not widely supported at the moment. No known attack currently target these ciphers.

So even though DSS is the last AESGCM on the list, it's still preferable to not using GCM. It's regular (SSL Labs, Chrome etc) name is TLS_DHE_DSS_WITH_AES_128_GCM_SHA256  (see https://www.openssl.org/docs/apps/ciphers.html), there's not a single http client using that in TLS_DHE_DSS_WITH_AES_128_GCM_SHA256. So I'll remove this.